doc/source/whatsnew/v0.19.0.txt

```diff
@@ -1333,6 +1333,7 @@ Bug Fixes
 - Bug in ``pd.merge()`` may raise ``TypeError`` if input datetime-like has other unit than ``ns`` (:issue:`13389`)
 
 - Bug in ``HDFStore``/``read_hdf()`` discarded ``DatetimeIndex.name`` if ``tz`` was set (:issue:`13884`)
+- Bug in ``pd.get_dummies`` returning dummy-encoded columns as floats, rather than integers (:issue:`8725`)
```

Comment:
not a bug fix more of an API change (not implemented before) ; I would make a sub section to highlite